tests/core/config/checkers/test_task_config_checker.py

```diff
@@ -311,3 +311,38 @@ def mock_func():
         Config.check()
         assert len(Config._collector.errors) == 0
         assert len(Config._collector.warnings) == 2
+
+    def test_check_required_property(self, caplog):
```

Comment:
Usually for checker tests, we also test the case where there's no error.

I would also configure a task with `"task_type": True, "required_key_1": "sth", "required_key_2": "sth_2"` to make sure it passes